provisioning/base/resources.tf

```diff
@@ -0,0 +1,60 @@
+provider "aws" {
+  region     = "us-east-1"
+}
+
+variable "environment" {
+  type = string
+  default = "qa"
+  description = "The name of the environnment (qa, production). This controls the name of lambda and the env vars loaded."
+
+  validation {
+    condition     = contains(["qa", "production"], var.environment)
+    error_message = "The environment must be 'qa' or 'production'."
+  }
+}
+
+# Package the app as a zip:
+data "archive_file" "lambda_zip" {
+  type        = "zip"
+  output_path = "${path.module}/dist.zip"
+  source_dir  = "../../"
+  excludes    = [".git", ".terraform", "provisioning"]
+}
+
+# Upload the zipped app to S3:
+resource "aws_s3_bucket_object" "uploaded_zip" {
+  bucket = "nypl-travis-builds-${var.environment}"
```

Comment:
Let's put these in `nypl-github-actions-builds-..`

Reply:
Fixed